solhint.js

```diff
@@ -17,7 +17,7 @@ function init() {
     .option('-f, --formatter [name]', 'report formatter name (stylish, table, tap, unix)')
     .option(
       '-w, --max-warnings [maxWarningsNumber]',
-      'number of warnings to trigger nonzero exit code'
+      'number of allowed warnings to not trigger nonzero exit code'
```

Comment:
Isn't something in the fashion of `number of allowed warnings` actually easier to read?

Reply:
I left the original comment about a nonzero exit code to avoid being too opinionated, but I do think your description should be enough.